kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableConfig.java

```diff
@@ -69,6 +70,45 @@ public class TableConfig<T> {
      * All the valid properties that can be set on the table.
      */
     private static final HashMap<String, TableConfig> validProperties = new HashMap<>();
+
+    /**
+     * This table property is used to track the enablement of the inCommitTimestamps.
```

Comment:
```suggestion
     * This table property is used to track the enablement of the {@code inCommitTimestamps}.
```

Reply:
Also a one line description of what this does? Basically this text from the protocol rfc. `When enabled, commit metadata includes a monotonically increasing timestamp that allows for reliable TIMESTAMP AS OF time travel even if filesystem operations change a commit file's modification timestamp.`